### ISSUE TITLE
Pagination contain weird characters (Â» and Â«)

### ISSUE THREAD

**Issue by ve3** [zurb/foundation-sites#8374]
I was download Foundation from download page Complete section.
Look into the file foundation.min.css
These classes.
```css
.pagination-previous.disabled::before, .pagination-previous a::before{
    content: "Â«";
}
.pagination-next.disabled::after, .pagination-next a::after{
    content: "Â»";
}
```
It contain some weird characters which is not exists in foundation.css (non-minified). Are your minifier have some problem or something?

**Comment by rafibomb** [comment]
That is weird! It could be the minification because the Sass looks right: https://github.com/zurb/foundation-sites/blob/develop/scss/components/_pagination.scss#L147

**Comment by ve3** [comment]
@rafibomb  It is not just this issue. https://github.com/zurb/foundation-sites/issues/8242

"Complete" and "Custom" download section are broken!

**Comment by gakimball** [comment]
This might be fixed by 0008fde4c5f3a10744c187fd05bb45c4286ba8b5, but the customizer has a known issue where it's pulling an older build of the framework. We're working on a solution to the customizer over here: #8400

**Comment by Owlbertz** [comment]
@ve3 Any changes that this got fixed in the mean time?
